backend/base/views/order_views.py

```diff
@@ -77,7 +79,7 @@ def get(self, request, pk, format=None):
         user = request.user
 
         try:
-            order = Order.objects.get(id=pk)
+            order = get_object_or_404(Order, id=pk)
             if user.is_staff or order.user == user:
```

Comment:
tutaj get_object_or_404 jest niepotrzebne jeśli robisz to w try/except
pozostała logika zwracająca response do przeniesienia poza try/except

Reply:
done